kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -245,7 +245,7 @@
       // link to where the 'back' button should go
       immersivePageRoute: {
         type: Object,
-        required: false,
+        required: true,
```

Comment:
I don't think this should be required, since it's only needed if the `immersivePage` prop is true. You will potentially see a lot of prop validation errors on non-immersive pages.